backend/logger/fileHandler.py

```diff
@@ -0,0 +1,80 @@
+import os
+import uuid
+import datetime
+import atexit
+
+
+class FileHandler:
+    """
+    In charge of handling files for the logger. This class will open, close,
+    and name files for the logger. Responsible for storing logs in safe place.
+    In addition, it will pass file handles to different programs.
+    """
+
+    def __init__(self, file_name):
+        self.file_name = file_name
+        self.full_name = file_name
+        self.file_handle = None
+
+        atexit.register(self.closeFile)
+
+    def createName(self):
+        """Create and return the name of the file.
+        The format is filename-uid-date
+
+        Returns:
+        name (str) : the name of the file
+        """
+        date = datetime.date.today().strftime("%b-%d-%Y")
```

Comment:
Is this date object standardized? I.e. is it always going to return three letters for the month and a 2 digit number for the day? Else we might want to use an 8 digit int instead to represent the date e.g. `01252021`

Reply:
The format at the moment is Month-Day-Year with the month written out for better readability and to avoid confusion between month and day. For example today's date would be "Jan-25-2021". Why do you think an 8-digit format is better in our case?

Reply:
oh I realize now that the month, day and year are separated with dashes, I was worried about parsing bugs. But it should be easy to parse out the date if we ever have to, so no need to change anything